go-controller/pkg/ovn/ovn.go

```diff
@@ -68,6 +68,9 @@ type namespaceInfo struct {
 	hybridOverlayExternalGW net.IP
 	hybridOverlayVTEP       net.IP
 
+	routingExternalGWs []net.IP
+	podExternalRoutes  map[string]map[string]string
```

Comment:
add comments explaining what these fields are, and particularly what the 3 strings in the `map[string]map[string]string` are

Reply:
done